src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/schema_validation.py

```diff
@@ -275,8 +275,11 @@
     int
 )]
 
+# Pipeline Params
+
 TARGET_SCHEMA = {
     Optional("path"): Or(str, int, TARGET_LIST_SCHEMA),
+    Optional("tags"): {str: object},
```

Comment:
Currently, the schema would allow empty strings as keys as well.
The value of the tags should be string as well right?

I think we can replace the definition to:
```python
Optional("tags"): { And(str, Regex(r"\\A.{1,128}\\Z")): And(str, Regex(r"\\A.{0,256}\\Z")) }
```
According to the [Tag Restrictions](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions) it can be any Unicode char for the key, as long as it is less than 128 chars long.
For the value, it can be any Unicode string up to 256 chars long.

Reply:
Ill add this.